### PR TITLE
NanoUI Licensing and Contributors

### LINKED ISSUE #3411: Please credit/ping upstream
I noticed you guys are maintaining your own fork of NanoUI (which looks pretty good by the way). However, I also can't help but notice that a lot of changes and code have come from upstream (/tg/station), and come without any sort of credit attached. I'd love for you to ping me when you downstream my refactors/changes so I can see what you did with it and assist with any issues, and I'd also love for you to notify me of any changes you make so that upstream can benefit.

I see it as a win/win. I'm sure there was no malicious intent, but it was a bit jarring to me to see my code duplicated line for line, or my Nanotrasen SVG logo all without any attribution. Thanks!

### PR DESCRIPTION
**Pull request by tigercat2000**
Creates the contributors section of the NanoUI README.md. Specifies
the NanoUI licensing and gives credit where appropriate to the parties
involved with NanoUI's code.

Probably resolves #3411

### CODE REVIEW

**Comment by TheDZD** [comment]
@neersighted Just curious (since @tigercat2000 is too anti-social to ask), how does tgstation/-tg-station#14933 affect the code that was ported prior to the PR for the licensing switch, if at all?

**Comment by neersighted** [comment]
Consider all code before that point AGPL. The only reason for the license switch was to allow other codebases to more easily import tgui and potentially use it outside of SS13. I retain rights to tgui and if MIT doesn't work for you I'm sure something can be worked out. But I would suggest just adopting the MIT license version of tgui as it will all be MIT going forward.

File: nano/README.md [hunk 1]
```diff
@@ -8,6 +8,35 @@ template engine.
...
+ - -tg-station/Neersighted
+  - Special thanks goes to Neersighted for creating most of the newest and best features
+    of NanoUI. Everything between using the Gulp build system to switching to LESS styles
+    has been directly adapated and/or copied from his original work for -tg-station and
+    their UI system, tgui.
+
+    Systems ported/adapted from -tg-station and Neersighted include, but are not limited
+    to:
+
+    - NanoUI's JavaScript controlling the "fancy" borderless window mode.
```
Comment (neersighted) [inline nano/README.md]: A little more detailed than needed but sure, I don't mind.
I would also add baystation to this list for completeness, especially Faerdan and PsiOmegaDelta.

**Comment by Fox-McCloud** [comment]
@tigercat2000 

Going to go with @neersighted's suggestion for including Psi and so?

**Comment by Fox-McCloud** [comment]
Going to go ahead and merge this, as @tigercat2000  is going to be absent for  quite a while; when they return further additions/changes can be made.

**Comment by Spacemanspark** [comment]
Wait what.
Why is @tigercat2000 leaving us.
Nooooooooo.

**Pull request merged**